src/style.css

```diff
@@ -1,3 +1,69 @@
 body {
+  font-family: Roboto, sans-serif;
+  font-size: 12px;
+
+  --hover-color: #00acdc;
+}
+
+.container {
+  display: flex;
+  height: 60px;
+  align-items: center;
+  align-content: space-around;
   margin: 0;
 }
+
+.logo {
+  display: block;
+  width: 40px;
+  height: 40px;
+}
+
+.nav__list {
+  display: flex;
+  width: 1200px;
+  justify-content: flex-end;
+  list-style: none;
+  padding: 0;
+  margin-left: 50px;
+}
+
+.nav__item {
+  display: block;
+}
+
+.nav__link {
+  color: #000;
+  text-decoration: none;
+  text-transform: uppercase;
+  font-size: 12px;
+  line-height: 14px;
+  padding: 5px;
+  position: relative;
+}
+
+.nav__link.is-active {
+  display: inline-block;
```

Comment:
The '.nav__link.is-active' class has a 'display: inline-block;' property which might not be necessary. If you're trying to override a previous 'display' value, ensure it's required; otherwise, it can be removed.